test/parallel/test-buffer-readuintle.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+// testing basic functionality of readUIntLE()
+
+const buf = Buffer.from([42, 84, 168, 127]);
+const result = buf.readUIntLE(2);
+
+assert.strictEqual(result, 168);
+
+assert.throws(
+      () => { buf.readUIntLE(5); }, RangeError);
+
+assert.doesNotThrow(
+      () => { assert.strictEqual(buf.readUIntLE(5, 0, true), undefined); },
```

Comment:
As mentioned above, if you could make this indented by just two characters, that would likely be preferable. Again, sorry our linter doesn't yet report that.